components/ui/__tests__/button.test.tsx

```diff
@@ -168,16 +152,9 @@ describe("Button", () => {
 
   describe("OutlineButton", () => {
     it("renders correctly", () => {
-      const { container } = render(<OutlineButton>test button</OutlineButton>);
+      render(<OutlineButton>test button</OutlineButton>);
 
-      expect(container.firstChild).toMatchInlineSnapshot(`
-        <button
-          class="inline-flex flex-row px-4 py-2 text-sm rounded-md items-center justify-center border border-transparent font-medium shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2 text-blue-500 border-blue-500 hover:bg-blue-100 focus:ring-blue-500 disabled:cursor-not-allowed disabled:opacity-75"
-          type="button"
-        >
-          test button
-        </button>
-      `);
+      expect(screen.getByText("test button")).toBeInTheDocument();
```

Comment:
I think we can keep the .toMatchInlineSnapshot() usage here since we're actually testing the styling for this component and the snapshot result is very small and readable.